modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="configuring-linuxptp-services-as-boundary-clock_{context}"]
+= Configuring Linuxptp services as Boundary Clock
+
+The PTP Operator adds the `PtpConfig.ptp.openshift.io` custom resource definition (CRD) to {product-title}.
+You can configure the linuxptp services (ptp4l, phc2sys) by creating a `PtpConfig` custom resource (CR) object.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator.
+
+.Procedure
+
+. Create the following `PtpConfig` CR, and then save the YAML in the `<name>-ptp-config.yaml` file. Replace `<name>` with the name for this configuration.
```

Comment:
I think you could simplify this somewhat: "Create the following `PtpConfig` CR, and then save the YAML in the `boundary-clock-ptp-config.yaml` file." I don't think we need to be exhaustive in naming the yaml file, since we apply it and forget about it basically.